recipes/checksumdir/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Where does this license come from? Upstream lists this one, which is included in the tar ball: https://github.com/cakepietoast/checksumdir/blob/master/LICENSE.txt

Reply:
Yeah, sorry about that, I'll remove it and reference the `LICENSE.txt` within the tarball instead

Reply:
No worries. Generally, Licenses from the tar ball should be used and if they’re not included you can add them manually and make a PR upstream to include them with the next release.